### ISSUE TITLE
Textarea in the questionnaire lacks label information 

### ISSUE THREAD

**Issue by peterk** [mysociety/fixmystreet#3944]
The textarea in the questionnaire lacks a label which may impact accessibility (evaluated with the MS Accessibility insights browser extension). Could be solved by either adding an explicit label or connect the preceding paragraph element via [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby).

https://github.com/mysociety/fixmystreet/blob/0540f7950d0a44a6575e94e2ff8a31cc970820b1/templates/web/base/questionnaire/index.html#L75

**Comment by dracos** [comment]
Hi, thanks for spotting this and letting us know, I've got a PR in to fix it :)
